proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -213,7 +221,6 @@ public void done(ServerPing result, Throwable error)
                     @Override
                     public void done(ProxyPingEvent pingResult, Throwable error)
                     {
-                        BungeeCord.getInstance().getConnectionThrottle().unthrottle( getAddress().getAddress() );
```

Comment:
You're never unthrottling, this will break Minecraft versions which double ping,